src/Otp/GoogleAuthenticator.php

```diff
@@ -50,6 +50,11 @@ public static function getKeyUri($type, $label, $secret, $counter = null, $optio
             throw new \InvalidArgumentException('Label has to be one or more printable characters');
         }
 
+        $parts = explode(':', $label);
```

Comment:
It's a check to see if there is at most one `:` in the label, therefor `strpos` is not a sufficient solution. I have changed this to `substr_count`.